packages/ember-views/lib/mixins/view_support.js

```diff
@@ -478,7 +478,7 @@ export default Mixin.create({
       { id: 'ember-views.dispatching-modify-property', until: '3.0.0' }
     );
 
-    if (!this.scheduledRevalidation || this._dispatching) {
+    if ((!this.scheduledRevalidation && !this.isDestroying) || this._dispatching) {
```

Comment:
this boolean doesn't seem correct, why would we schedule if isDestroying but we are this._dispatching? seems like isDestroying should just exit early altogether.